.rubocop_todo.yml

```diff
@@ -0,0 +1,326 @@
+# This configuration was generated by `rubocop --auto-gen-config`
+# on 2015-04-27 23:58:52 +0200 using RuboCop version 0.30.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
```

Comment:
I thought about this and have one question: Are we going to follow the style guide 100 percent? This will mean backwards incompatible with Ruby <= 1.8, is that ok?